components/LanguageSelector.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { SUPPORTED_LANGUAGES } from '../i18n';
 import { I18nManager, View } from 'react-native';
 import { Picker } from './ui/Picker';
-import AsyncStorage from '@react-native-async-storage/async-storage';
+import { setDataToStorage } from '../shared/storage';
```

Comment:
I would probably expect exporting Storage as a class, rather than individual methods. These methods can on the storage class.

Eg:
``` 
import Storage from '../shared/storage';
Storage.setItem(...)
```

Reply:
Similar changes can be done everywhere in the code